code/go/0chain.net/smartcontract/dbs/event/updater.go

```diff
@@ -59,6 +63,39 @@ func CreateBuilder(table string, idColumn string, idValues interface{}) (b *Upda
 // If expression is set, then it will be used instead of default, e. g. expr = "table.column =table.column + t.column ",
 // which means that current value of column will be increased, so *UPDATE table SET table.column =table.column + t.column* will be generated instead
 func (b *UpdateBuilder) AddUpdate(column string, values interface{}, expr ...string) *UpdateBuilder {
+	b.addToSets(column, values, expr...)
+	b.addToUnnests(column, values)
+	b.addToValues(values)
+	return b
+}
+
+// AddCondition Add ANDed condition to the update query 
+func (b *UpdateBuilder) AddCondition(column string, values interface{}) *UpdateBuilder {
```

Comment:
what is this special conditions, if it is a part of composite key we should add it to the ID logic, not as additional clause